packages/cojson/package.json

```diff
@@ -35,6 +35,9 @@
     "neverthrow": "^7.0.1",
     "queueueue": "^4.1.2"
   },
+  "peerDependencies": {
+    "@opentelemetry/api": "^1.0.0"
+  },
```

Comment:
Not sure about having this as peer dependency.

If we are importing it and the user doesn't install open telemetry doesn't it break the build?

Reply:
Having it as a peer dependency ensures that the users installs the version they need/want and avoid inconsistencies as only one version is going to be installed - in general all instrumented libraries mark it as a peer dep. Some of them - like nextjs - make it also optional by also shipping a pre bundled version that is used if the user does not install another one explicitly - i guess to remove the friction when using a package manager that does not install peer deps automatically.

Reply:
yep. and some other mark it as a dependency, like https://github.com/vercel/ai/blob/311b17306f9aada804622e319270094347be004c/packages/ai/package.json

Reply:
If you feel confident that works it's ok for me.

Was just worried by the fact that peerDependencies haven't been correctly implemented by some package managers over time and that someone might incur in some issues.

But this might be something related to the past, honestly I don't know how the situation is nowadays.

Reply:
let's see if it still breaks people's installs